rts/lra/test/basic/pom.xml

```diff
@@ -56,4 +57,18 @@
 
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <includes>
+            <include>**/*Test*.java</include>
```

Comment:
Why have you added this specification? How does it relate to your PR. There are a number of basic tests that don't have Test in their name (all of the integration tests for example), In fact the only thing that will now get tested is the new test you have added in this PR. In fact if you did want to be consistent with the naming then, since the failsafe plugin is for integration testing, `<include>**/*IT*.java</include>` would have been more natural. But why do you need explicitly tell the build which classes to use, all of the tests in the `src/test` directory need to be ran.

Reply:
I think it might not be needed - I can see some config here (https://github.com/jbosstm/narayana/blob/master/rts/lra/pom.xml#L134), I will try without it